app/views/reviews/_index.html.erb

```diff
@@ -4,4 +4,7 @@
     <%= review.description %>
     <%= review.user.email %>
   </div>
+  <% if review.user == current_user %>
```

Comment:
consider using a model behavior to determine if the current user can edit the record.

```erb
<% if review.editable_by?(current_user) -%>
```